apps/parsley/src/components/LogWindow/index.tsx

```diff
@@ -21,6 +22,7 @@ const LogWindow: React.FC = () => {
 
   return (
     <Container data-cy="log-window">
+      <SectionsFeatureModal />
```

Comment:
we can init this in the context and lazy load it irt my other comments.

Reply:
I was able to lazy load the modal within this component and can see from the network traffic that it loads the graphic conditionally.